### DISCUSSION TITLE
Total fraction metabolized by CYPs with large molecule model

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1043] [Questions & Problems]
Hi all,

I am building a model for a large and a small molecule using the protein model. The small molecule is metabolized by two CYPs, and I would like to know the total fraction metabolized by each CYPs. When using the small molecule model, there is an observer for this, but it seems that the observer is not available with the protein model.
I tried then to find the observer in a small molecule model in MoBi, so that I could load it in the large molecule model, but I am not sure I was able to locate the right observer.
Could someone tell me the procedure to have this observer also for the large molecule model? Or is there an easier way to get this output? I am using PK-Sim 9.
(By the way, is there a reason why this observer is not available by default with the protein model? Since this model can also be used for small molecules).

Thank you in advance for your help!

Donato

**Comment by prvmalik** [comment]
Hi try outputting the concentration of the respective metabolites as molar unit in the liver intracellular space to get a first idea. Obviously the error here will be if you have the enzyme present in tissues beyond the liver, but you get a preliminary idea.

**Reply by teutonicod** [reply]
Thanks @prvmalik , indeed it is an option, but with a full observer it would be more precise, since as you mentioned I would get the metabolism from all the tissue. This is particularly important for me since I would need to calibrate the total fraction metabolized to the in vitro measures for each enzyme.
For this reason I was wondering if it would be possible just to copy the observer from the small molecule model.

Donato

**Comment by Yuri05** [comment]
Hi Donato,

Total fraction metabolized observer is defined in **large molecule** models in exactly the same way as in **small molecule** models.
So if you don't see it in your large molecules simulation - it might be due to other reasons.
![grafik](https://user-images.githubusercontent.com/25061876/145824185-a627f053-c888-44b4-9430-b6d8284a26b9.png)

**Reply by ytg922** [reply]
Hi Yuri05,
 why I don't see the compartment Organism in the define settings and run function of the simulation interface?
